### ISSUE TITLE
Sync Failed

### ISSUE THREAD

**Issue by hf-hf** [jellyfin/jellyfin-kodi#849]
Describe the bug
Resource Sync failed，reinstalling Kodi and sync again still cannot run，I think the log may indicate that a certain resource is missing the VideoRangeType field. After clearing some resources and rescanning the media library, it still hasn't returned to normal.

Logs
```
2024-04-07 16:00:03.338 T:45264    info <general>: JELLYFIN.jellyfin_kodi.objects.tvshows -> DEBUG::jellyfin_kodi\objects\tvshows.py:299 EpisodeId be1c02b26f5f1863ccdbee0fea57fe3f not found
2024-04-07 16:00:03.350 T:45264    info <general>: JELLYFIN.jellyfin_kodi.database -> ERROR::jellyfin_kodi\database\__init__.py:165 type: <class 'KeyError'> value: 'VideoRangeType'
2024-04-07 16:00:03.353 T:45264    info <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi\database\__init__.py:169 [jellyfin] 1 rows updated.
2024-04-07 16:00:03.356 T:45264    info <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi\database\__init__.py:172 ---<[ database: jellyfin ] 1847219069936
2024-04-07 16:00:03.359 T:45264    info <general>: JELLYFIN.jellyfin_kodi.database -> ERROR::jellyfin_kodi\database\__init__.py:165 type: <class 'KeyError'> value: 'VideoRangeType'
2024-04-07 16:00:03.361 T:45264    info <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi\database\__init__.py:169 [video] 21 rows updated.
2024-04-07 16:00:03.371 T:45264    info <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi\database\__init__.py:172 ---<[ database: video ] 1847219071616
2024-04-07 16:00:03.380 T:13560    info <general>: Loading skin file: DialogConfirm.xml, load type: KEEP_IN_MEMORY
2024-04-07 16:00:05.315 T:13560 warning <general>: CGUIWindowManager - CGUIWindowManager::HandleAction - ignoring action 107, because topmost modal dialog closing animation is running
2024-04-07 16:00:05.335 T:45264    info <general>: JELLYFIN.jellyfin_kodi.full_sync -> ERROR::jellyfin_kodi\full_sync.py:274 full sync exited unexpectedly
2024-04-07 16:00:05.343 T:45264    info <general>: JELLYFIN.jellyfin_kodi.full_sync -> ERROR::jellyfin_kodi\full_sync.py:275 'VideoRangeType'
                                                   Traceback (most recent call last):
                                                     File "jellyfin_kodi\full_sync.py", line 260, in process_library
                                                       media[library['CollectionType']](library)
                                                     File "jellyfin_kodi\helper\wrapper.py", line 41, in wrapper
                                                       result = func(self, dialog=dialog, *args, **kwargs)
                                                     File "jellyfin_kodi\full_sync.py", line 358, in tvshows
                                                       obj.episode(episode)
                                                     File "jellyfin_kodi\helper\wrapper.py", line 65, in wrapper
                                                       return func(*args, **kwargs)
                                                     File "jellyfin_kodi\helper\wrapper.py", line 77, in wrapper
                                                       return func(self, item, e_item=e_item, *args, **kwargs)
                                                     File "jellyfin_kodi\objects\tvshows.py", line 325, in episode
                                                       obj['Video'] = API.video_streams(obj['Video'] or [], obj['Container'])
                                                     File "jellyfin_kodi\helper\api.py", line 79, in video_streams
                                                       elif track['VideoRangeType'] in ["HDR10", "HDR10Plus"]:
                                                   KeyError: 'VideoRangeType'
                                                   
2024-04-07 16:00:05.346 T:45264    info <general>: JELLYFIN.jellyfin_kodi.helper.utils -> DEBUG::jellyfin_kodi\helper\utils.py:56 --[ window clear: jellyfin_sync ]
2024-04-07 16:00:05.352 T:45264    info <general>: JELLYFIN.jellyfin_kodi.full_sync -> INFO::jellyfin_kodi\full_sync.py:603 --<[ fullsync ]
2024-04-07 16:00:05.360 T:45264    info <general>: JELLYFIN.jellyfin_kodi.library -> ERROR::jellyfin_kodi\library.py:532 'VideoRangeType'
                                                   Traceback (most recent call last):
                                                     File "jellyfin_kodi\library.py", line 530, in add_library
                                                       sync.libraries(library_id, update)
                                                     File "jellyfin_kodi\full_sync.py", line 109, in libraries
                                                       self.start()
                                                     File "jellyfin_kodi\full_sync.py", line 193, in start
                                                       self.process_library(library)
                                                     File "jellyfin_kodi\full_sync.py", line 260, in process_library
                                                       media[library['CollectionType']](library)
                                                     File "jellyfin_kodi\helper\wrapper.py", line 41, in wrapper
                                                       result = func(self, dialog=dialog, *args, **kwargs)
                                                     File "jellyfin_kodi\full_sync.py", line 358, in tvshows
                                                       obj.episode(episode)
                                                     File "jellyfin_kodi\helper\wrapper.py", line 65, in wrapper
                                                       return func(*args, **kwargs)
                                                     File "jellyfin_kodi\helper\wrapper.py", line 77, in wrapper
                                                       return func(self, item, e_item=e_item, *args, **kwargs)
                                                     File "jellyfin_kodi\objects\tvshows.py", line 325, in episode
                                                       obj['Video'] = API.video_streams(obj['Video'] or [], obj['Container'])
                                                     File "jellyfin_kodi\helper\api.py", line 79, in video_streams
                                                       elif track['VideoRangeType'] in ["HDR10", "HDR10Plus"]:
                                                   KeyError: 'VideoRangeType'
```
System (please complete the following information):
Jellyfin Version: 10.8.0
Kodi Version: Kodi v21 (Omega)

**Comment by mcarlton00** [comment]
Looks like fallout from #804.  Was this a movie library or something else?  I didn't notice any errors from my testing, but it's always possible I missed something.

**Comment by gurg1e** [comment]
I needed to resync my whole library and i'm getting a failure too. It just sits at 0% stuck on the first episode of the fist show in my library. 

Jellyfin 10.8.13
JfK 1.0.1
Kodi 19.5

```` 2024-04-09 10:24:09.517 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.objects.tvshows -> INFO::jellyfin_kodi/objects/tvshows.py:311 EpisodeId 1 missing from kodi. repairing the entry.
2024-04-09 10:24:09.575 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.objects.tvshows -> DEBUG::jellyfin_kodi/objects/tvshows.py:404 Setting episode pathParentId, episode 45b2ab24b3a9d76867094b5a4a4d8144, title Countdown, pathId 4, pathParentId None
2024-04-09 10:24:09.583 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.objects.tvshows -> DEBUG::jellyfin_kodi/objects/tvshows.py:407 ADD episode [4/2] 45b2ab24b3a9d76867094b5a4a4d8144: Countdown
2024-04-09 10:24:09.592 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> ERROR::jellyfin_kodi/database/__init__.py:165 type: <class 'sqlite3.OperationalError'> value: table streamdetails has no column named strHdrType
2024-04-09 10:24:09.596 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi/database/__init__.py:169 [jellyfin] 2 rows updated.
2024-04-09 10:24:09.605 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi/database/__init__.py:172 ---<[ database: jellyfin ] 68548680
2024-04-09 10:24:09.611 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> ERROR::jellyfin_kodi/database/__init__.py:165 type: <class 'sqlite3.OperationalError'> value: table streamdetails has no column named strHdrType
2024-04-09 10:24:09.615 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi/database/__init__.py:169 [video] 81 rows updated.
2024-04-09 10:24:09.629 T:3746     INFO <general>: JELLYFIN.jellyfin_kodi.database -> DEBUG::jellyfin_kodi/database/__init__.py:172 ---<[ database: video ] 70197544 ````

**Comment by oddstr13** [comment]
@gurg1e You're seeing a different but related issue

**Comment by hf-hf** [comment]
Can I use VideoRange without VideoRangeType?

**Comment by oddstr13** [comment]
@hf-hf Should be fixed in 1.0.2
